composer.lock

```diff
@@ -66,26 +66,26 @@
         },
         {
             "name": "brick/math",
-            "version": "0.9.3",
+            "version": "0.10.2",
```

Comment:
Don't commit any changes to the `composer.lock` file. We only commit changes when we update the `composer.json` file.